lib/rails/html/sanitizer.rb

```diff
@@ -112,6 +112,7 @@ def sanitize(html, options = {})
           # No duck typing, Loofah ensures subclass of Loofah::Scrubber
           loofah_fragment.scrub!(scrubber)
         elsif allowed_tags(options) || allowed_attributes(options)
+          remove_xpaths(loofah_fragment, XPATHS_TO_REMOVE)
```

Comment:
Followup to my comment on #27. I'd be happier if we're conditionally removing the xpaths, like this:
```ruby
@permit_scrubber.tags = allowed_tags(options)
remove_xpaths(loofah_fragment, XPATHS_TO_REMOVE) if @permit_scrubber.tags
```